.github/workflows/triage.yml

```diff
@@ -0,0 +1,13 @@
+name: Issue triage
+on:
+  issues:
+    types: [opened]
+jobs:
+  build:
+    name: Triage
+    runs-on: ubuntu-16.04
```

Comment:
@gauravchaudhari02 
Why Ubuntu when our entire system is being designed around CentOS?

Reply:
👋🏼 @ypise, GitHub hosted runners do not have an option to specify CentOS and support only Ubuntu Linux, Microsoft Windows, and macOS for now.
More info - https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners